OWML.ModLoader/Owo.cs

```diff
@@ -37,19 +43,36 @@ public void LoadMods()
                 _console.WriteLine("Verbose mode is enabled");
                 Application.logMessageReceived += OnLogMessageReceived;
             }
-            var mods = _modFinder.GetMods();
-            foreach (var modData in mods)
+            var normalMods = _modFinder.GetMods().Where(mod => !mod.Manifest.PriorityLoad).ToList();
+            var sortedNormal = _sorter.SortMods(normalMods);
+
+            var priorityMods = _modFinder.GetMods().Where(mod => mod.Manifest.PriorityLoad).ToList();
+            var sortedPriority = _sorter.SortMods(priorityMods);
+
+            var modNames = _modFinder.GetMods().Where(mod => mod.Config.Enabled).Select(mod => mod.Manifest.UniqueName);
+
+            var sortedMods = Enumerable.Concat(sortedPriority, sortedNormal);
+
+            foreach (var mod in sortedMods)
             {
-                var modType = LoadMod(modData);
+                foreach (var dep in mod.Manifest.Dependencies)
```

Comment:
```suggestion
                foreach (var dependency in mod.Manifest.Dependencies)
```
Unnecessary abbreviations make code less readable below